.github/workflows/build.yml

```diff
@@ -118,6 +117,9 @@ jobs:
         with:
           images: ghcr.io/espressosystems/cape/wallet
 
+      - name: Generate base image
+        run: bash bin/build-docker-base
```

Comment:
This is gonna actually be `wallet-checkout/bin/build-docker-base` for weird permissions reasons.

Reply:
@Ancient123 seems to work now.